frontend/web/components/Dateline.tsx

```diff
@@ -1,36 +1,15 @@
 import { Component } from 'react';
-import { getCookie } from '../lib/cookie';
-import dateformat from 'dateformat';
-
-const dtFormatGU = (date: Date, edition: string): string => {
-    let datetimeFormatted: string = dateformat(date, 'ddd d mmm yyyy HH:MM Z');
-    if (edition === 'UK') {
-        datetimeFormatted = datetimeFormatted.replace('GMT+0100', 'BST');
-    }
-    return datetimeFormatted;
-};
-
-const defaultEdition = 'UK';
 
 class Dateline extends Component<
-    { capiDate: Date },
-    { capiDate: Date; edition: string }
+    { dateDisplay: string },
+    { dateDisplay: string }
 > {
-    constructor(props: { capiDate: Date }) {
+    constructor(props: { dateDisplay: string }) {
         super(props);
-        this.state = { capiDate: this.props.capiDate, edition: defaultEdition };
-    }
-    public componentDidMount() {
-        const cookieEdition = getCookie('GU_EDITION');
-        if (cookieEdition && this.state.edition !== cookieEdition) {
-            this.setState({
-                capiDate: this.state.capiDate,
-                edition: cookieEdition,
-            });
-        }
+        this.state = { dateDisplay: this.props.dateDisplay };
     }
     public render() {
-        return dtFormatGU(this.state.capiDate, this.state.edition);
+        return this.state.dateDisplay;
```

Comment:
It looks like we no longer need `state`. We can pull `dateDisplay` straight off the `props`. Also we can refactor this component as a [function component](https://reactjs.org/docs/components-and-props.html#function-and-class-components).

Reply:
Good point, will make a commit for it.